src/libaktualizr/package_manager/fetcher_test.cc

```diff
@@ -56,7 +64,10 @@ void test_pause(const Uptane::Target& target) {
 
   std::shared_ptr<INvStorage> storage(new SQLStorage(config.storage, false));
   auto http = std::make_shared<HttpClient>();
-  Uptane::Fetcher f(config, storage, http, progress_cb);
+
+  auto pacman = std::make_shared<PKGMGR_TYPE>(config.pacman, storage, nullptr, http);
```

Comment:
We try to avoid having tests that run differently depending on CMake flags, so that we can be confident we test everything when the test suite runs with everything `=on`. The CMake flags are only here to add functionality, not to configure it.

In this case, `test_pause()` could have the package manager type as a template parameter, or could take a `std::function` factory.

Reply:
good idea. thanks.